src/simulator_core/entities/assets/pipe.py

```diff
@@ -127,17 +128,17 @@ def _get_heat_transfer_coefficient(self, esdl_asset: EsdlAssetObject) -> float:
         diameters, heat_coefficients = get_thermal_conductivity_table(esdl_asset=esdl_asset)
         if diameters:
             # Create a numpy array of the diameters and heat coefficients
-            diameters = np.array(diameters)
-            heat_coefficients = np.array(heat_coefficients)
+            diameters_np = np.array(diameters)
```

Comment:
Waarom heb je dit moeten doen?

Reply:
Foutmelding van Mypip die je krijgt als je type overschrijft

src\simulator_core\entities\assets\pipe.py:131: error: Incompatible types in assignment (expression has type "ndarray[Any, dtype[Any]]", variable has type "list[float]")  [assignment]
src\simulator_core\entities\assets\pipe.py:136: error: Argument "inner_diameter" to "calculate_inverse_heat_transfer_coefficient" has incompatible type "list[float]"; expected "ndarray[Any, Any]"  [arg-type]
src\simulator_core\entities\assets\pipe.py:137: error: Argument "outer_diameter" to "calculate_inverse_heat_transfer_coefficient" has incompatible type "list[float]"; expected "ndarray[Any, Any]"  [arg-type]